source/contracts/Augur.sol

```diff
@@ -132,4 +133,9 @@ contract Augur is Controlled {
         TokensTransferred(_universe, msg.sender, _from, _to, _value);
         return true;
     }
+
+    function getProtectedTokens() internal returns (address[]) {
```

Comment:
It would be good to specify storage in the return type: `returns (address[] memory)`